app/javascript/controllers/auto_citation_controller.js

```diff
@@ -10,9 +10,13 @@ export default class extends Controller {
 
     this.updateDisplay()
 
-    this.displayDefault(this.switchTarget.checked)
+    // If the manualTarget matches the autoTarget, then display the auto.
+    const showAuto = this.manualTarget.value == this.autoTarget.value
```

Comment:
Should `==` be `===` here?

Reply:
Does `[1] == true` ?

Reply:
I am not a JavaScript interpreter. :laughing: